src/IIIFManifestParser.php

```diff
@@ -85,7 +87,12 @@ public function getTileSources($manifest_url) {
 
     try {
       // Request the manifest.
-      $manifest_response = $this->httpClient->get($manifest_url);
+      // $manifest_response = $this->httpClient->get($manifest_url);
+      $manifest_response = $this->httpClient->request('GET', $manifest_url, [
+        'headers' => [
+          'Authorization' => 'Bearer ' . $access_token,
+        ],
+      ]);
```

Comment:
Not sure that this makes sense in the case `$access_token` was _not_ passed, as we would generate an `Authorization` header of just `Bearer ` (or possibly `Bearer null`?).

Really, if we want to be sure to always have a token here, why not just generate it here?